stdlib/builtin/regexp.rbs

```diff
@@ -1,123 +1,359 @@
-class Regexp < Object
+class Regexp
+  # Alias for Regexp.new
+  #
+  def self.compile: (String arg0, ?untyped options, ?String kcode) -> Regexp
```

Comment:
I think it's better to have good argument names instead of `arg0` or something like that. I usually find the _good names_ from argslist.